src/main/java/io/vertx/ext/sql/ResultSet.java

```diff
@@ -142,11 +143,26 @@ public ResultSet setNext(ResultSet next) {
    * @return  the rows represented as JSON object instances
    */
   public List<JsonObject> getRows() {
+    return getRows(false);
+  }
+
+  /**
+   * Get the rows - each row represented as a JsonObject where the keys are the column names and the values are
+   * the column values.
+   * <p>
+   * Beware that it's legal for a query result in SQL to contain duplicate column names, in which case one will
+   * overwrite the other if using this method. If that's the case use {@link #getResults} instead.
+   *
+   * @param caseInsensitive - treat column names as case insensitive, i.e.: FoO equals foo equals FOO
```

Comment:
How's the Json row serialized to `Buffer` or `String`? With column name as returned by the database right? Perhaps we should add a note about this in the Javadoc. In case the user takes it as-is before sending over http or whatnot.

Reply:
I've added some docs regarding encoding

Reply:
:+1: